build.gradle.kts

```diff
@@ -6,7 +6,7 @@ import org.jetbrains.kotlin.gradle.dsl.JvmTarget
 import org.jetbrains.kotlin.gradle.dsl.KotlinVersion
 
 plugins {
-    `kotlin-dsl`
```

Comment:
I wonder if we can remove some Kotlin lang version hackery below.

Reply:
I'm sure that will be possible - maybe in a follow up rather than this PR